packages/runtime/lib/runtime.js

```diff
@@ -739,7 +739,7 @@ class Runtime extends EventEmitter {
     // Setup ITC
     service[kITC] = new ITC({ port: service })
     service[kITC].listen()
-    service[kITC].handle('getServiceMeta', this.#getServiceMeta.bind(this))
+    service[kITC].handle('getServiceMeta', this.getServiceMeta.bind(this))
```

Comment:
Why do you need it? Do you need to pass connection strings from one service to another?

Reply:
Marco used that method in tests now.

Reply:
Why this hanlder os defined here and not in the itc file and it doesn't call a stackable.getServiceMeta?

Reply:
cc @ShogunPanda